src/org/usfirst/frc4904/robot/RobotMap.java

```diff
@@ -57,55 +69,79 @@
 		
 		public static class PCM {}
 		
-		public static class Pneumatics {}
+		public static class Pneumatics {
+			public static int solenoidUp = 0;
+			public static int solenoidDown = 1;
+		}
+	}
+	
+	public static class Constant {
+		public static class Chassis {
+			public static double TURN_P = 0.02;
+			public static double TURN_I = 0.001;
+			public static double TURN_D = 0.3;
+			public static final double MAX_DEGREES_PER_SECOND = 120;
+			public static final double WHEEL_RADIUS_INCHES = 8;
+			public static final double WHEEL_PULSES_PER_REVOLUTION = 1024;
+		}
+	}
+	
+	public static class HumanInput {
+		public static class Driver {
+			public static CustomXbox xbox;
+		}
+		
+		public static class Operator {
+			public static CustomJoystick stick;
+		}
 	}
 	
```

Comment:
At this level, make a new static class called `Metrics`, and put in `WHEEL_PULSES_PER_REVOLUTION`